plugins/out_es/es.c

```diff
@@ -62,8 +62,8 @@ static flb_sds_t add_aws_auth(struct flb_http_client *c,
     flb_http_add_header(c, "User-Agent", 10, "aws-fluent-bit-plugin", 21);
 
     signature = flb_signv4_do(c, FLB_TRUE, FLB_TRUE, time(NULL),
-                              ctx->aws_region, "es",
-                              0,
+                              ctx->aws_region, ctx->aws_service_name,
+                              S3_MODE_SIGNED_PAYLOAD, NULL,
                               ctx->aws_provider);
```

Comment:
Don't we want to add the AWS unsigned headers in `es` as well? Otherwise there is no point in adding the AWS_Service_Name field to this plugin I think?

Reply:
Goood point